enclave/crypto/cert.c

```diff
@@ -738,16 +740,36 @@ oe_result_t oe_cert_chain_read_pem(
         OE_RAISE(OE_INVALID_PARAMETER);
 
     /* Must have pem_size-1 non-zero characters followed by zero-terminator */
-    if (strnlen((const char*)pem_data, pem_size) != pem_size - 1)
-        OE_RAISE(OE_INVALID_PARAMETER);
+    // Note: the above comment is incorrect. There is no trailing zero,
+    // but mbedtls_x509_crt_parse() requires a trailing zero
```

Comment:
You can probably remove the above command and just say "mbedtls_x509_crt_parse() requires a trailing zero"

Reply:
Agreed